docs/source/algorithms/AlignComponents-v1.rst

```diff
@@ -9,16 +9,36 @@
 Description
 -----------
 
-This algorithm will take a calibration in the form of a
-:ref:`diffractioncalibration
-workspace<DiffractionCalibrationWorkspace>` from the output of *for
-example* :ref:`algm-GetDetOffsetsMultiPeaks` or
-:ref:`algm-CalibrateRectangularDetectors` and minimize the difference
-between the DIFC of the instrument and calibration workspace by moving
-and rotating instrument components.
+This algorithm will take a table of peak-center positions (TOF units) for every pixel,
+and minimize the following quantity by moving and rotating instrument components.
 
-The resulting calibrated geometry can be exported by
-:ref:`algm-ExportGeometry`.
+.. math:: \sum_i^{N_d}\sum_j^{M_i} \frac{|d_{i,j} - d^*_j|}{d^*_j}
```

Comment:
This obscures the input time-of-flight of the peak center. It may be clearer to change the equation to
```suggestion
.. math:: \sum_i^{N_d}\sum_j^{M_i} \frac{|DIFC_i *TOF_{i,j} - d^{exp}_j|}{d^*_j}
```